.github/workflows/build-macos-installer.yml

```diff
@@ -108,8 +108,42 @@ jobs:
         cd ../build_scripts
         sh build_macos.sh
 
+    - name: Create installer version number
+      id: version_number
+      run: |
+        . ./activate
+        echo "::set-output name=CHIA_INSTALLER_VERSION::$(chia version)"
+
     - name: Upload MacOS artifacts
       uses: actions/upload-artifact@v2
       with:
         name: Chia-Installer-on-MacOS-10.15-dmg
         path: ${{ github.workspace }}/build_scripts/final_installer/
+
+    - name: Install AWS CLI
+      run: |
+        curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+        sudo installer -pkg AWSCLIV2.pkg -target /
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.INSTALLER_UPLOAD_KEY }}
+        aws-secret-access-key: ${{ secrets.INSTALLER_UPLOAD_SECRET }}
+        aws-region: us-west-2
+
+    - name: Create Checksums
+      run: |
+          ls
+          shasum -a 256 ${{ github.workspace }}/build_scripts/final_installer/Chia-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}.dmg > ${{ github.workspace }}/build_scripts/final_installer/Chia-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}.dmg.checksum
+
+    - name: Upload to s3
+      run: |
+          aws s3 cp ${{ github.workspace }}/build_scripts/final_installer/Chia-${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}.dmg s3://download-chia-net/builds/
```

Comment:
Is this going to be the same filename as the builds generated in Azure? I think we're going to be fighting over which Mac job (GHA or Azure) gets the actual spot in S3 depending on which finishes last

Reply:
GHA is invariably last and that's bad as we want to ship the Azure/Mojave build.